src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -385,6 +385,14 @@ class BinaryCompactObject : public DomainCreator<3> {
         "Use projective scaling on the frustal cloak."};
   };
 
+  struct FrustumSphericity {
+    using group = EnvelopingCube;
+    static std::string name() { return "Sphericity"; }
+    using type = double;
+    static constexpr Options::String help = {
+        "Sphericity of the enveloping cube."};
```

Comment:
Could you add a bit of info here? Help strings that just repeat the option name are not super helpful :D
You can also add `lower_bound` and `upper_bound` functions.